initrd/bin/seal-hotpkey

```diff
@@ -49,26 +57,35 @@ mount_boot
 counter_value=1
 
 enable_usb
-if ! libremkey_hotp_verification info ; then
-  echo "Insert your Librem Key and press Enter to configure it"
+if ! hotp_verification info ; then
+  echo "Insert your $HOTPKEY_BRANDING and press Enter to configure it"
   read
-  if ! libremkey_hotp_verification info ; then
+  if ! hotp_verification info ; then
     # don't leak key on failure
     shred -n 10 -z -u "$HOTP_SECRET" 2> /dev/null
-    die "Unable to find Librem Key"
+    die "Unable to find $HOTPKEY_BRANDING"
   fi
 fi
 
+# Set HOTP USB Security Dongle branding based on VID
+if [ ! $(lsusb | grep -q "20a0:") ]; then
+	HOTPKEY_BRANDING="Nitrokey"
+elif [ ! $(lsusb | grep -q "316d:") ]; then
```

Comment:
not sure why, but this ends up showing my Librem Key as a Nitrokey.
using:
`elif $(lsusb | grep -q "316d:" ; then`
works correctly however

Reply:
The negation is used because grep returns 0 if it found something.

Older Librem Keys had the same VID as the Nitrokeys because of a mistake done when building the firmware for it, see https://github.com/osresearch/heads/pull/761#issuecomment-651598181

I could not think of a way to prevent that :(

So I am wondering, what is the return value of `lsusb | grep -q "316d:"` for you? If it is `0` the patch should work as intended. Does it?

Reply:
In other words first batches of the Librem Key had incorrectly set VID:PID same as Nitrokey Pro has, hence the confusion.
Perhaps they have changed the USB names only back then (might not show up in the `lsusb`, but maybe `dmesg`) - that would already help to differentiate between models, otherwise I do not see how to make it possible.

Reply:
> So I am wondering, what is the return value of lsusb | grep -q "316d:" for you? If it is 0 the patch should work as intended. Does it?

0, and no the patch does not work correctly with my LK. It identifies as a NK without the change I mentioned above

Reply:
pushed #781 to fix